docs/gen-1/red-blue/main-glitchless/beginner-route/2021/lass/README.md

```diff
@@ -458,7 +458,7 @@ NOTE: From this point on you have 2 revives which means deaths aren't as scary,
 - EQ x3
 - Elixer on Weezing
   - if Koga used x attack use Bubblebeam till you die
-  - if you didn’t die from koga you can still try to get red bar by following the HP strats for the [backups](https://pastebin.com/HgyuMvyr). Note that this can be risky
+  - if you didn’t die from koga you can still try to get red bar by following the HP strats for the [backups](https://pastebin.com/2DBBnkmf). Note that this can be risky
```

Comment:
why this change? this pastebin is old and doesnt even have boom strats

Reply:
This PR has old commits in it and doesn't represent the real changes being made yet.